packages/media/src/resourceList.ts

```diff
@@ -19,7 +19,7 @@
  */
 import type { ResourceId } from '@googleforcreators/types';
 
-interface ResourceCacheEntry {
+export interface ResourceCacheEntry {
   url: string;
   type: 'cached' | 'fullsize';
```

Comment:
Should we make this a proper enum? Would remove the need for types in the `resourceList.test.ts` suite.